### PR TITLE
feat: support download image

### PR DESCRIPTION
**Pull request by pangda0xff**

### CODE REVIEW

**Comment by alswl** [comment]
Cool pull request, can you provide some screenshots to show the effect of this feature please?

File: yuque2markdown.py [hunk 1]
```diff
@@ -77,14 +87,45 @@ def extract_repos(repo_dir, output, toc):
...
-            output_path = os.path.join(output_dir_path, sanitized_title + '.md')
+            if download_image:
+                html = download_image_replace_path(
+                    output_dir_path, sanitized_title, html)
+
+            output_path = os.path.join(
+                output_dir_path, sanitized_title + '.md')
             f = open(output_path, 'w')
-            f.write(pretty_md(md(html)))
+            f.write(pretty_md(md(html, heading_style='ATX')))
```
Comment (alswl) [inline yuque2markdown.py:97]: Maybe ATS can be a options? Or we can keep the default behavior.

File: yuque2markdown.py [hunk 2]
```diff
@@ -77,14 +87,45 @@ def extract_repos(repo_dir, output, toc):
...
 
         last_sanitized_title = sanitized_title
         last_level = current_level
 
 
+def download_image_replace_path(output_dir_path, sanitized_title, html):
```
Comment (alswl) [inline yuque2markdown.py:103]: Maybe `download_images_and_patch_html`

File: yuque2markdown.py [hunk 3]
```diff
@@ -102,9 +143,12 @@ def pretty_md(text: str) -> str:
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Convert Yuque doc to markdown')
+    parser = argparse.ArgumentParser(
+        description='Convert Yuque doc to markdown')
     parser.add_argument('lakebook', help='Lakebook file')
     parser.add_argument('output', help='Output directory')
+    parser.add_argument(
+        '--download-image', help='If download images to local', action='store_true')
```
Comment (alswl) [inline yuque2markdown.py:151]: `If download images to local` to `Download images to local`

**Comment by zhuxining** [comment]
Run successfully, the image download in the attachment folder, such as screenshot

image

**Comment by alswl** [comment]
Merge is completed, thanks for you contribution.

I will fix some description later.

**Pull request merged**